Description: ARM Support Patch
 Original patch from Ubuntu, modified to include ARM64
Author: Danie Lintott <daniel@serverb.co.uk>
Origin: Ubuntu
---
Index: dynamips/stable/utils.h
===================================================================
--- dynamips.orig/stable/utils.h	2013-10-29 11:44:22.292749962 +0000
+++ dynamips/stable/utils.h	2013-10-29 12:20:28.417518086 +0000
@@ -55,6 +55,8 @@
 #define ARCH_BYTE_ORDER ARCH_LITTLE_ENDIAN
 #elif defined(__ia64__)
 #define ARCH_BYTE_ORDER ARCH_LITTLE_ENDIAN
+#elif defined(__arm__) || defined (__aarch64__)
+#define ARCH_BYTE_ORDER ARCH_LITTLE_ENDIAN
 #endif
 
 #ifndef ARCH_BYTE_ORDER
